azext_edge/tests/edge/support/create_bundle/test_mq_int.py

```diff
@@ -0,0 +1,118 @@
+# coding=utf-8
+# ----------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License file in the project root for license information.
+# ----------------------------------------------------------------------------------------------
+
+import pytest
+from knack.log import get_logger
+from azext_edge.edge.common import OpsServiceType
+from .helpers import check_name, check_non_custom_file_objs, get_file_map, run_bundle_command
+
+logger = get_logger(__name__)
+
+
+@pytest.mark.parametrize("mq_traces", [False, True])
+def test_create_bundle_mq(init_setup, tracked_files, mq_traces):
+    """Test for ensuring file names and content. ONLY CHECKS mq."""
+    mq_traces = True
+
+    ops_service = OpsServiceType.mq.value
+    command = f"az iot ops support create-bundle --mq-traces {mq_traces} --ops-service {ops_service}"
+    walk_result = run_bundle_command(command=command, tracked_files=tracked_files)
+    file_map = get_file_map(walk_result, ops_service, mq_traces=mq_traces)
+    traces = file_map.pop("trace", {})
+    assert "diagnostic_metrics" in file_map
+
+    check_mq_types(file_map)
+
+    expected_file_objs = {
```

Comment:
I like the idea of this data structure where there is expected resource types per service.

Reply:
may want to make these into typed dict or some other class to give more structure

Reply:
future pr for this